lang/ghc/Portfile

```diff
@@ -134,14 +134,29 @@ compiler.library_path \
 # binaries into ${prefix}, not DESTDIR.  Work around this by setting
 # --prefix=${destroot}${prefix} and not setting DESTDIR
 
-if {![variant_isset "bootstrap"]} {
+if { [variant_isset "nobootstrap"] } {
     set bootstrap_dir ${destroot}${prefix}
 } else {
     set bootstrap_dir ${workpath}/bootstrap
 
     pre-configure {
         xinstall -m 0755 -d ${bootstrap_dir}
+
+        # effectively run `port select --set sphinx py37-sphinx` by hand
+        # so that the build finds the sphinx binaries
+        set sphinx_fd [open "${prefix}/etc/select/sphinx/base" r]
+        while { [gets ${sphinx_fd} sphinx_base_line] >= 0 } {
+            # create versionless symbolic links within ${bootstrap_dir}/bin,
+            # which is added to ${PATH} during the build
+            ln -s \
+                ${prefix}/${sphinx_base_line}-${python3_version} \
+                ${bootstrap_dir}/${sphinx_base_line}
+        }
+        close ${sphinx_fd}
```

Comment:
@kencu 
> port contents py37-sphinx to find the name of it's binary

Yes, did that. I actually overkilled it by just adding symlinks from `sphinx_select` by hand, show here.